tests/rustdoc-ui/c-help.stdout

```diff
@@ -5,6 +5,7 @@
     -C         debug-assertions=val -- explicitly enable the `cfg(debug_assertions)` directive
     -C                debuginfo=val -- debug info emission level (0 = no debug info, 1 = line tables only, 2 = full debug info with variable and type information; default: 0)
     -C default-linker-libraries=val -- allow the linker to link its default libraries (default: no)
+    -C                  dlltool=val -- import library generation tool (windows-gnu only)
```

Comment:
Can you replace `(windows-gnu only)` with `(ignored except when targeting windows-gnu)`. I think that's more clear.